packages/admin/admin/src/form/FinalFormInput.tsx

```diff
@@ -38,9 +37,9 @@ export function FinalFormInput({
                 {...input}
                 {...props}
```

Comment:
Please add `required` back here, as you removed it from `props` above

```suggestion
                {...props}
                required={required}
```